lib/view/bbcode/Parser.class.php

```diff
@@ -586,34 +506,74 @@ function bb_nobold($arguments = array()) {
 		return $return;
 	}
 
+	/**
+	 * Bold
+	 *
+	 * @example [b]Bold[/b]
+	 */
 	function bb_b() {
```

Comment:
Deze functie wordt ge-overload in CsrBB, zelfde geldt voor `bb_i`, `bb_u`, etc. Beter deze verwijderen.